examples/burial-form/MilitaryServiceHistory.jsx

```diff
@@ -1,18 +1,71 @@
+/* eslint-disable @typescript-eslint/no-unsafe-return */
 import React from 'react';
 import {
+  ArrayField,
   DateField,
   Page,
   TextField,
   SelectField,
+  DebuggerView
 } from '@department-of-veterans-affairs/va-forms-system-core';
 import { useFormikContext } from 'formik';
 import { isBeforeDate } from './utils';
-import { VaButton } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+
+const ARRAY_FIELD_PROPS = {
+  dateRange: {
+    to: undefined,
+    from: undefined,
+  },
+  serviceBranch: undefined,
+  rank: undefined,
+  serviceNumber: undefined,
+  placeOfEntry: undefined,
+  placeOfSeparation: undefined,
+}
+
+const FieldArrayTemplate = ({data, index}) => {
```

Comment:
Can you explain why this is a template inside of the JSX file? Are users going to have to create this same type of object/component? If so can we document above to explain.

Reply:
Yes, in order to have it populate with data from an individual form, this is the markup that gets displayed per array item.

Reply:
TY :+1: